src/battle_util.c

```diff
@@ -133,11 +138,15 @@ static const u16 sRolePlayBannedAbilities[] =
     ABILITY_SHIELDS_DOWN,
     ABILITY_DISGUISE,
     ABILITY_RKS_SYSTEM,
-    ABILITY_BATTLE_BOND,
     ABILITY_POWER_CONSTRUCT,
+    ABILITY_BATTLE_BOND,
     ABILITY_ICE_FACE,
-    ABILITY_HUNGER_SWITCH,
```

Comment:
Is this correct? Smogon [says that Role Play can't copy Hunger Switch](<https://www.smogon.com/dex/sv/moves/role-play/>).